config/localization/en/app.json

```diff
@@ -1526,31 +1526,20 @@
       "YOURE_IN": "You're in {LEAGUE}"
     },
     "VAULTS": {
-      "VAULT_DESCRIPTION": "Vaults provide automated liquidity on all dYdX markets. Vaults aim for a market-neutral position by quoting both sides of the book. P&L will vary based on market conditions and there's a risk of losing some or all of the USDC deposited.",
-      "LP_VAULT": "LP Vault",
       "VAULT": "Vault",
-      "ASSET_VAULT": "{ASSET} Vault",
-      "VAULTS": "Vaults",
-      "VAULT_DETAILS": "Vault Details",
-      "VAULT_FAQS": "Vault FAQs",
-      "VAULT_BALANCE": "Vault Balance",
-      "VAULT_POSITION": "Vault Position",
-      "VAULT_LEVERAGE": "Vault Leverage",
-      "VAULT_ALL_TIME_PNL": "All-time Vault P&L",
+      "YOUR_ALL_TIME_PNL": "Your All-time P&L",
       "VAULT_THIRTY_DAY_APR": "30d APR",
-      "VAULTS_OVERVIEW": "Vaults Overview",
-      "VAULT_DOCS": "Vault Docs",
-      "ALL_VAULTS": "All Vaults",
-      "MY_VAULTS": "My Vaults",
-      "YOUR_BALANCE": "Your Balance",
-      "SEARCH_VAULTS": "Search Vaults",
-      "YOUR_LP_VAULT_BALANCE": "Your LP Vault Balance",
+      "VAULT_THIRTY_DAY_PNL": "30d P&L",
+      "YOUR_VAULT_BALANCE": "Your Vault Balance",
       "VIEW_VAULT": "View Vault",
       "VAULT_EQUITY": "Vault Equity",
       "VAULT_PNL": "Vault P&L",
-      "DEPOSITS_AND_WITHDRAWALS": "Deposits & Withdrawals",
-      "VIEW_ALL_VAULTS": "View All Vaults",
-      "YOU_HAVE_NO_VAULT_DEPOSITS": "You have no vault deposits."
+      "YOUR_DEPOSITS_AND_WITHDRAWALS": "Your Deposits & Withdrawals",
+      "YOU_HAVE_NO_VAULT_DEPOSITS": "You have no vault deposits.",
+      "7_DAYS": "7d",
+      "30_DAYS": "30d",
+      "90_DAYS": "90d",
```

Comment:
nit: unsure if we need to translate these strings, or if we can just use [getStringsForTimeInterval](https://github.com/dydxprotocol/v4-web/blob/main/src/lib/timeUtils.ts#L27)

Reply:
if we do need to translate, mild preference for something like [X_MINUTES_LOWERCASED](https://github.com/dydxprotocol/v4-localization/blob/main/config/localization/en/app.json#L35) which is more reusable